luajava/src/main/java/party/iroiro/luajava/Lua.java

```diff
@@ -759,6 +759,19 @@ public interface Lua extends AutoCloseable {
      */
     void createTable(int nArr, int nRec);
 
+    /**
+     * Creates a new empty table and pushes it onto the stack
+     * 
+     * Does the equivalent to createTable(0, 0)
+     * 
+     * <p>
+     * The new table but without pre-allocation.
+     * </p>
+     */
+    default void newTable() {
```

Comment:
Instead of a default method (which causes trouble for tests on Android), please put the implementation in `AbstractLua`.